### PR TITLE
Update release notes unity

### PR DESCRIPTION
**Pull request by shanmydell**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/1386 |

# Checklist:

- [X] Have you run a grammar and spell checks against your submission?
- [X] Have you tested the changes locally?
- [X] Have you tested whether the hyperlinks are working properly?
- [X] Did you add the examples wherever applicable?
- [X] Have you added high-resolution images?

### CODE REVIEW

**Review by AkshaySainiDell — DISMISSED** [review]

File: content/docs/csidriver/release/unity.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ description: Release notes for Unity XT CSI driver
...
 | fsGroupPolicy may not work as expected without root privileges for NFS only [https://github.com/kubernetes/examples/issues/260](https://github.com/kubernetes/examples/issues/260) | To get the desired behavior set “RootClientEnabled” = “true” in the storage class parameter |
-| Controller publish is taking too long to complete/ Health monitoring is causing Unity array to panic by opening multiple sessions | Disable Volume health monitoring on the node and keep it only at the controller level.|
+| Controller publish is taking too long to complete/ Health monitoring is causing Unity array to panic by opening multiple sessions | Disable Volume health monitoring on the node and keep it only at the controller level. Refer [here](https://dell.github.io/csm-docs/docs/csidriver/features/unity/#volume-health-monitoring) for more information about enabling/disabling volume health monitoring|
```
Comment (gallacher) [inline content/docs/csidriver/release/unity.md:39]: "... As a workaround, Disable Volume health... " - Disable should be disable and Volume should be volume. What other information can be provided besides "Controller publish is taking too long?". What about things that would be observed in the logs? Under what circumstances would this be seen?

Comment (shanmydell) [reply content/docs/csidriver/release/unity.md:39]: @gallacher :  Health monitoring is causing Unity array to panic by opening multiple sessions - This is the reason why it has been seen, it is also added

Comment (gallacher) [reply content/docs/csidriver/release/unity.md]: @shanmydell - what errors are seen in the logs? What's observed in k8s? Please provide more details.

**Review by chimanjain — APPROVED** [review]
LGTM

File: content/docs/csidriver/release/unity.md (hunk 1 above)
Comment (chimanjain) [reply content/docs/csidriver/release/unity.md]: Thanks @shanmydell for mentioning the error we are receiving in k8s logs, reason and the roundabout to circumvent the issue.

**Review by shefali-malhotra — APPROVED** [review]

**Pull request merged**